File: URI_Python_UFRR/1013.py
linha1 = input().split(" ")

a, b, c = linha1

maiorAB = (int(a)+int(b)+abs(int(a)-int(b)))/2
if maiorAB == int(a):
	if int(a) > int(c):
		print('{} eh o maior'.format(int(a)))
	else: print('{} eh o maior'.format(int(c)))

else:
	if int(b) > int(c):
		print('{} eh o maior'.format(int(b) ))
	else: print('{} eh o maior'.format(int(c) ))

File: URI_C_UFRR/1013.c
#include <stdio.h>
int main(){

    int a,b,c,maiorAB;
    scanf("%i %i %i",&a,&b,&c);
    maiorAB = (a+b+abs(a-b))/2;
    if(maiorAB==a){
        if(a>c){
            printf("%i eh o maior\n",a);
        }else{
            printf("%i eh o maior\n",c);
        }
    }else{
        if(b>c){
            printf("%i eh o maior\n",b);
        }else{
            printf("%i eh o maior\n",c);
        }
    }

    return 0;
}

